[DOCS] Fix formatting issue in cardinality-aggregation.asciidoc (#98175)
elastic/elasticsearch: docs/reference/aggregations/metrics/cardinality-aggregation.asciidoc

--------------------------------------------------
// TEST[setup:sales]
<1> Documents without a value in the `tag` field will fall into the same bucket as documents that have the value `N/A`.

> ==== Execution Hint

Author: ```suggestion
==== Execution hint
```

--------------------------------------------------
// TEST[setup:sales]
<1> Documents without a value in the `tag` field will fall into the same bucket as documents that have the value `N/A`.

==== Execution hint

> There are different mechanisms by which cardinality aggregations can be run:

Reviewer: ```suggestion
You can run cardinality aggregations using different mechanisms:
```
